### ISSUE TITLE
Github Action

### ISSUE THREAD

**Issue by CharMixer** [github-changelog-generator/github-changelog-generator#748]
Thanks a lot for the work done by this community, It saved me a lot of time.

To be able to integrate it with a new project, I've created a github action which uses the docker image already created by @ferrarimarco. I thought that this could be useful for someone else.

It can be found here: https://github.com/marketplace/actions/automatic-changelog-generator
I'm quite new to Github Actions, so there might be some thing which can be done better.

It's currently used by a new project which can be found here https://github.com/OpenSentry
You're more than welcome to add this to the "Projects using this library" list.

**Comment by skywinder** [comment]
Thanks. It's a good idea to add it as Action.

 I'm also new with actions and didn't have time to figure out, how to work with it. Let's have a look. If you have an idea, how to set up it in repo or make a quick readme addition based on your experience, how to setup it in your repo - I would be happy to merge it!

btw, you can add your project, since it's a wiki page and everyone can edit it: https://github.com/github-changelog-generator/github-changelog-generator/wiki/Projects-using-Github-Changelog-Generator

**Comment by skywinder** [comment]
I see your action, but don't see any kind of code or instrucrtions how to setup it:
https://github.com/OpenSentry/idp/actions?query=workflow%3AChangelog

It would be cool, if you can add this instruction for other users. But it's definetly cool thing to automate!

**Comment by CharMixer** [comment]
Sorry for my late response.

I will update the action with some more insights on how to set it up.
I just finished a new flow, which uses a PR-flow instead of a direct master push.
This allows protected branches better.

I'm already getting some stars for this action, so I will of course try to get this into a more stable version. Currenly, only a subset of params is supported and testing has been minimal.

For anyone dropping by until I get the readme updated, all you have to do is add a file in your git project in `/.github/workflows/changelog.yml`. This is where the configuration from https://github.com/marketplace/actions/automatic-changelog-generator goes. Github actions will take care of the rest. Trigger the action with any event you have defined and watch it run under the action tab in your github repo.

**Comment by skywinder** [comment]
@CharMixer  thanks for the update. sounds very promising! Please, let me know if you want to integrate this feature. I would also love to add you as a member of this group if you are willing to improve this project! 👍

**Comment by CharMixer** [comment]
Thanks for the offer.
I'm not sure what you mean by "integrate this feature", other than adding it to the readme.  It would probably be better to add you to my project, if you would like to add some new parameters to it for instance.

**Comment by skywinder** [comment]
@CharMixer 
You're right. I mean that we can add it as an example to this repo. So people can copy-paste it to their own. It would be nice to make the readme section about this too.

Or let me know if you will be done this with your repo. So I will make the same in this repo.
